docs/web3-shh.rst

```diff
@@ -63,7 +63,7 @@ Returns
 -------
 
 
-``String`` - The version of the current whisper running.
+``Promise`` returns ``String`` - The version of the current whisper running.
```

Comment:
I would like this syntax more: ```Promise<String>```  🤔

Reply:
I would as well but the web3-eth describes it like this so i wanted to keep it consistent 

https://web3js.readthedocs.io/en/1.0/web3-eth.html#getcoinbase